.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+version: 2.1
+orbs:
+  node: circleci/node@5.0.2
+jobs:
+  build:
+    executor: node/default
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          command: npm install
+          name: Install npm
+      - run:
+          command: sudo npm install -g ts-node
```

Comment:
think this theoretically isn't necessary, since it should be part of `npm install`. Does that not work?

Reply:
try deleting this again, think should work